cmd/archeio/app/clientip.go

```diff
@@ -47,7 +47,6 @@ func getClientIP(r *http.Request) (netip.Addr, error) {
 	// precede <client-ip>,<load-balancer-ip> in this header.
 	// The preceding IP addresses might contain other characters, including spaces.
 	rawXFwdFor := r.Header.Get("X-Forwarded-For")
```

Comment:
note: `Get` gets the first header value, I think cloud run flattens this header, but we will find out soon enough with some e2e tests.

Reply:
yes, `curl -v -H "Forwarded: aaa" -H "X-Forwarded-For: 8.8.8.8" -H "X-Forwarded-For: 127.0.0.1" https://registry-sandbox.k8s.io/v2/pause/blobs/sha256:da86e6ba6ca197bf6bc5e9d900febd906b133eaa4750e6bed647b0fbe50ed43aa` => logs show correct IP detected still.